pants-plugins/release/rules.py

```diff
@@ -215,20 +230,35 @@ async def inject_package_fields(
     )
 
     version: str = extracted_version.value
-    if version.endswith("dev") and version[-4] != "-":
+    is_dev = "dev" in version
+    if is_dev and "-dev" not in version:
         # nfpm parses this into version[-version_prerelease][+version_metadata]
-        # that dash is required to be a valid semver version.
+        # that dash is required to be a valid semver version (3.9dev => 3.9-dev).
         version = version.replace("dev", "-dev")
 
+    # this is specific to distro-version (EL8, EL9, Ubuntu Focal, Ubuntu Jammy, ...)
+    next_release = await packagecloud_get_next_release(
+        PackageCloudNextReleaseRequest(
+            nfpm_arch=target[NfpmArchField].value,
+            distro_id="",  # TODO: add field for distro ID
```

Comment:
What is the todo for?

Reply:
To clarify, this line will end up being:
```suggestion
            distro_id=target[DistroIDField].value,
```